frontend-dev/src/components/AllIntegrations/GoogleSheet/GoogleSheetCommonFunc.js

```diff
@@ -258,12 +258,12 @@ export const handleAuthorize = (confTmp, selectedAuthType, setError, setIsLoadin
 
 }
 
-export const tokenHelper = async (grantToken, confTmp, setConf, selectedAuthType, authData, setAuthData, setIsLoading, setSnackbar) => {
+export const tokenHelper = async (authInfo, confTmp, setConf, selectedAuthType, authData, setAuthData, setIsLoading, setSnackbar) => {
   if (!selectedAuthType) {
     return
   }
   const tokenRequestParams = {}
-  tokenRequestParams.code = grantToken
+  tokenRequestParams.code = authInfo
```

Comment:
authInfo is an `object` and you are assigning an object in the tokenRequestParas.code